### ISSUE TITLE
SSL/Cert Problems -I think I borked my install

### ISSUE THREAD

**Issue by neuman1812** [Jackett/Jackett#1580]
Jackett version:  
Jackett Version 0.7.1588.0
Mono version (if not using Windows):  
Mono JIT compiler version 5.0.1.1 (2017-02/5077205 Thu May 25 10:07:22 UTC 2017)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
	TLS:           __thread
	SIGSEGV:       normal
	Notifications: epoll
	Architecture:  armel,vfp+hard
	Disabled:      none
	Misc:          softdebug 
	LLVM:          supported, not enabled.
	GC:            sgen (concurrent by default)

-------------------------------
After a reboot, All my trackers are coming back with the same error :+1: 
```
Feb 12 14:22:06 raspberry mono[2363]: 02-12 14:22:06 Error got NotImplemented/InternalServerError
Feb 12 14:22:06 raspberry rsyslogd-2007: action 'action 10' suspended, next retry is Fri Feb 12 14:23:36 2016 [v8.16.0 try http://www.rsyslog.com/e/2007 ]
Feb 12 14:22:06 raspberry mono[2363]: 02-12 14:22:06 Error request.Method: GET
Feb 12 14:22:06 raspberry mono[2363]: 02-12 14:22:06 Error request.Url: https://isohunt.to/latest.php?ihq=
Feb 12 14:22:06 raspberry mono[2363]: 02-12 14:22:06 Error request.Cookies: __cfduid=dd81a94c34c6cc20347ced3208762dd751499789804; ISOSESSID=9b6167a1b9495a545b19553e9b7549dd; _csrf=e9e7abd4e0b2f6225dd55c994ab73ecc2e36e1994e6886d23b12cb876939d308a%3A2%3A%7Bi%3A0%3Bs%3A5%3A%22_csrf%22%3Bi%3A1%3Bs%3A32%3A%22n58tdlgoh9j4Cf8jsZD15gDVxFjQRwd5%22%3B%7D
Feb 12 14:22:06 raspberry mono[2363]: 02-12 14:22:06 Error request.Referer:
Feb 12 14:22:06 raspberry mono[2363]: 02-12 14:22:06 Error request.RawPOSTDdata:
Feb 12 14:22:06 raspberry mono[2363]: 02-12 14:22:06 Error cookies:
Feb 12 14:22:06 raspberry mono[2363]: 02-12 14:22:06 Error headerString:
Feb 12 14:22:06 raspberry mono[2363]:  
Feb 12 14:22:06 raspberry mono[2363]: 02-12 14:22:06 Info Found 0 releases from Isohunt
Feb 12 14:22:06 raspberry mono[2363]: 02-12 14:22:06 Error Exception in test_indexer System.Exception: Found no results while trying to browse this tracker
Feb 12 14:22:06 raspberry mono[2363]:   at Jackett.Services.IndexerManagerService+<TestIndexer>c__async0.MoveNext () [0x000ef] in C:\projects\jackett\src\Jackett\Services\IndexerManagerService.cs:201
Feb 12 14:22:06 raspberry mono[2363]: --- End of stack trace from previous location where exception was thrown ---
Feb 12 14:22:06 raspberry mono[2363]:   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <a07d6bf484a54da2861691df910339b1>:0
Feb 12 14:22:06 raspberry mono[2363]:   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) [0x0003e] in <a07d6bf484a54da2861691df910339b1>:0
Feb 12 14:22:06 raspberry mono[2363]:   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) [0x00028] in <a07d6bf484a54da2861691df910339b1>:0
Feb 12 14:22:06 raspberry mono[2363]:   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) [0x00008] in <a07d6bf484a54da2861691df910339b1>:0
Feb 12 14:22:06 raspberry mono[2363]:   at System.Runtime.CompilerServices.TaskAwaiter.GetResult () [0x00000] in <a07d6bf484a54da2861691df910339b1>:0
Feb 12 14:22:06 raspberry mono[2363]:   at Jackett.Controllers.AdminController+<Test>c__async5.MoveNext () [0x000da] in C:\projects\jackett\src\Jackett\Controllers\AdminController.cs:269

```

I added --IgnoreSslErrors true to the startup and that seems to have resolved it,  but I'd like to fix it if possible.

**Comment by flightlevel** [comment]
Have you tried this ? https://github.com/Jackett/Jackett/issues/1113#issuecomment-302657339

Also try

```
wget https://curl.haxx.se/ca/cacert.pem
cert-sync --user ./cacert.pem
````


If it still happens run `mono --debug JackettConsole.exe -t -l` and post the FULL log, that snippet is pretty useless

**Comment by Speaktomenow** [comment]
I too am getting this error on latest Mono and Jackett. Upgraded to latest mono today for OSX ND suddenly all kinds of SSL cert errors. 
Added the cacert.pem and it hasn't fixed it.

**Comment by flightlevel** [comment]
Run the below commands and then copy and paste the output here

Try
`cert-sync`
and report version

`wget https://curl.haxx.se/ca/cacert.pem`
`cert-sync --user ./cacert.pem`

`csharp -e 'Console.WriteLine (new System.Net.WebClient ().DownloadString ("https://www.howsmyssl.com/").IndexOf ("1.2"))'`

Is this the installer you used
http://www.mono-project.com/download/#download-mac
5.0.1?

Then post the full Jackett log
ie
`mono --debug JackettConsole.exe -t -l`

**Comment by Speaktomenow** [comment]
Ok will do, that's the version of mono I installed yes.

**Comment by Speaktomenow** [comment]
I had already installed the latest cacert.pem file before posting, didn’t help.

I rolled back mono, re-installed Jackett and this has fixed the issue. Sonarr doesn’t seem to like older versions of mono, or at least it keeps telling me to upgrade.

Ignoring this prompt from Sonarr and just using the older version of mono seems to have got Jackett working without the SSL errors again.

Will have to wait and see what testing is like over next few days to be sure. If error persists will post logs - these were all deleted when uninstalling Jackett.

Thanks.
